src/main/java/org/mifos/integrationtest/config/KafkaConfig.java

```diff
@@ -0,0 +1,17 @@
+package org.mifos.integrationtest.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class KafkaConfig {
+
+    @Value("${kafka.brokers}")
```

Comment:
Assuming, this test cannot be run from outside the cluster. But I want to understand whether this can be a blocker to others as Kafka won't be available when this is run from a different server.